deploy/eck-stack/charts/eck-logstash/values.yaml

```diff
@@ -0,0 +1,60 @@
+---
+# Default values for eck-logstash.
+# This is a YAML-formatted file.
+
+# Overridable names of the Logstash resource.
+# By default, this is the Release name set for the chart,
+# followed by 'eck-logstash'.
+#
+# nameOverride will override the name of the Chart with the name set here,
+# so nameOverride: quickstart, would convert to '{{ Release.name }}-quickstart'
+#
+# nameOverride: "quickstart"
+#
+# fullnameOverride will override both the release name, and the chart name,
+# and will name the Logstash resource exactly as specified.
+#
+# fullnameOverride: "quickstart"
+
+# Version of Logstash.
+#
+version: 8.10.0-SNAPSHOT
+
+# Labels that will be applied to Logstash.
+#
+labels: {}
+
+# Annotations that will be applied to Logstash.
+#
+annotations: {}
+
+# Controlling the number of pods.
+# ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-logstash-scaling-logstash.html
+#
+count: 1
+
+# The logstash configuration, the ECK equivalent to logstash.yml
+# ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-logstash-configuration.html#k8s-logstash-configuring-logstash
+#
+config: {}
+
+# Set podTemplate to customize the pod used by Logstash
+# ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-customize-pods.html
+#
+podTemplate: {}
+
+monitoring: {}
+
+# The logstash pipelines, the ECK equivalent to pipelines.yml
+# ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-logstash-configuration.html#k8s-logstash-pipelines
+#
+pipelines: []
+
+# volumeClaimTemplates
+# ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-logstash-configuration.html#k8s-volume-claim-settings
+#
+volumeClaimTemplates: []
+
+elasticsearchRefs: []
+
+services: []
```

Comment:
`secureSettings` and `serviceAccountName` are also valid logstash options that we want to include here to allow override.

Reply:
`configRef` and `pipelinesRef` are also valid and both will need validation to ensure only one is configured.

Reply:
Where are documented these keys ? I don't see any of them in https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-logstash.html and its children

I would have added them if I had seen them

Reply:
@azro352 These are defined on the struct itself: 
https://github.com/elastic/cloud-on-k8s/blob/e94f7e1caa8ba70449c0bb8ef56eaac63a17dd25/pkg/apis/logstash/v1alpha1/logstash_types.go#L25-L96